src/TouristCards.js

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+
+function TouristInfoCards() {
```

Comment:
This component does the same thing three times - that works, but can you think of a sub-component you could make to make this less repetitive?